src/main/java/org/damap/base/conversion/AbstractTemplateExportSetup.java

```diff
@@ -53,18 +53,27 @@ protected void exportSetup(long dmpId) {
     costList = dmp.getCosts();
 
     // determine project leader/coordinator/principal investigator
-    Optional<Contributor> projectLeaderOpt =
+    List<Contributor> projectLeaderOpt =
         dmp.getContributorList().stream()
             .filter(
-                contributor -> contributor.getContributorRole() == EContributorRole.PROJECT_LEADER)
-            .findFirst();
-    if (projectLeaderOpt.isPresent())
-      projectCoordinator =
-          ContributorDOMapper.mapEntityToDO(projectLeaderOpt.get(), new ContributorDO());
-    else
+                contributor ->
+                    contributor.getContributorRole() == EContributorRole.PROJECT_LEADER
+                        || contributor.getContributorRole() == EContributorRole.PROJECT_COORDINATOR
+                        || contributor.getContributorRole()
+                            == EContributorRole.PRINCIPAL_INVESTIGATOR)
+            .toList();
+
+    if (!projectLeaderOpt.isEmpty()) {
+      projectCoordinator = new ArrayList<>();
+      for (Contributor contributor : projectLeaderOpt) {
+        projectCoordinator.add(ContributorDOMapper.mapEntityToDO(contributor, new ContributorDO()));
+      }
+    } else
       try {
         if (dmp.getProject() != null && dmp.getProject().getUniversityId() != null)
-          projectCoordinator = projectService.getProjectLeader(dmp.getProject().getUniversityId());
+          projectCoordinator =
+              List.of(projectService.getProjectLeader(dmp.getProject().getUniversityId()));
```

Comment:
The contributor returned from the project service should be assigned the role project leader, if he/she has no role assigned from the service.